frontend/riskofbias/robMetrics/DomainTable.js

```diff
@@ -58,17 +57,21 @@ class DomainTable extends Component {
                             <th>Domain #{domainIndex + 1}</th>
                             <th>{detailedView ? <DetailDomain object={domain} /> : domain.name}</th>
                             <th>
-                                <button
-                                    className="btn btn-info"
-                                    onClick={() => moveDomain(domainIndex, false)}>
-                                    <i className="fa fa-arrow-up"></i>
-                                </button>
+                                {domainCount > 1 && (
```

Comment:
this works fine, but elsewhere we use a ternary pattern for this instead of a chained logical. 

```javascript
{domainCount > 1 ? (
<button
        className="btn btn-info"
        onClick={() => moveDomain(domainIndex, false)}>
        <i className="fa fa-arrow-up"></i>
    </button>
) : null}
```

Suggest changing for internal consistency.